NEMP/mods.json

```diff
@@ -469,6 +469,26 @@
             "item": 0
         }
     },
+    "OpenSecurity-1.10.2": {
+        "name": "OpenSecurity",
+        "function": "CheckJenkins",
+        "active": true,
+        "jenkins": {
+            "url": "http://ci.pc-logix.com/job/OpenSecurity%201.10/lastSuccessfulBuild/api/json",
+            "regex": "OpenSecurity-(?P<mc>.+?)-(?P<version>.+?).jar$",
```

Comment:
`(?P<mc>.+?)` needs to be changed to `MC(?P<mc>.+?)` here, otherwise the bot tries to use the `MC1.10.2` list, which doesn't exist